main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/YoutubeAudioSourceManager.java

```diff
@@ -416,13 +416,13 @@ private boolean determineFailureReason(HttpInterface httpInterface, String video
   }
 
   private boolean determineFailureReasonFromStatus(String status, String reason, boolean mustExist) {
-    if ("fail".equals(status) || "ERROR".equals(status)) {
+    if ("fail".equals(status) || "ERROR".equals(status) || "UNPLAYABLE".equals(status)) {
       if (("This video does not exist.".equals(reason) || "This video is unavailable.".equals(reason)) && !mustExist) {
         return true;
       } else if (reason != null) {
         throw new FriendlyException(reason, COMMON, null);
       }
-    } else if ("ok".equals(status)) {
+    } else if ("ok".equals(status) || "OK".equals(status)) {
```

Comment:
Anything speaking against `String#equalsIgnoreCase` here?

Reply:
No, I just preferred the explicitness here

Reply:
Didn't mean to bully you into using it, just wanted to make sure you are aware of the option :)

Reply:
Also one thing to note on the commit that you pushed is that `"string".equals(someVariable)` has the advantage over `someVariable.equals("string")` in that it is null safe.

Reply:
Ahh wasn't aware of that, good to know 👍